### PR TITLE
13 stores index

### PR DESCRIPTION
**Pull request by amhursh**
Additions: Added basic stores index functionality

User Story: #13 - Stores Index

Concerns (code location):

Mentions to collaborators:

Notes(if applicable):

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -40,7 +40,7 @@ gem 'materialize-form'
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
-  gem 'pry'
+  gem 'pry-rails'
```
Comment (blsrofe) [inline Gemfile]: How is this different from regular pry?

File: app/views/welcome/index.html.erb [hunk 2]
```diff
@@ -7,64 +7,18 @@
...
 </div>
-<br />
-<br />
-<br />
 
-<%= image_tag "bury.gif" %>
-<%= image_tag "trash.gif" %>
+<div>
+  <%= link_to "View Stores", stores_path %>
 </div>
```
Comment (blsrofe) [inline app/views/welcome/index.html.erb]: Why are we taking out the carousel? I thought we were going to have a carousel of items?

File: spec/features/stores/user_can_visit_stores_index_spec.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    visit '/'
+
+    click_on 'View Stores'
+
+    expect(current_path).to eq stores_path
+
+    expect(page).to have_content first_store.name
+    expect(page).to have_content last_store.name
+  end
+end
```
Comment (blsrofe) [inline spec/features/stores/user_can_visit_stores_index_spec.rb:20]: This is great!

**Review by blsrofe — APPROVED** [review]

**Comment by blsrofe** [comment]
Great work. I can use your tests as a model for mine!

File: app/views/welcome/index.html.erb [hunk 4]
```diff
@@ -14,7 +14,7 @@
...
 
 <div>
-  <%= link_to 'All Stores', stores_path %>
+  <%= link_to 'View Stores', stores_path %>
```
Comment (mikeheft) [inline app/views/welcome/index.html.erb:17]: makes the readability better with the view instead of all. I like it.

**Comment by mikeheft** [comment]
Looks good. 👍

**Pull request merged**